_posts/en/newsletters/2020-12-16-newsletter.md

```diff
@@ -0,0 +1,346 @@
+  associated wallet and other software.  Note, the rc3 version of the
+  [signed binary][macos signed] will not run due to a problem with the
+  code signing tool.  The [unsigned version][macos unsigned] (which can
+  still be [verified][pgp verification] with PGP) should run if opened
+  using the right-click (or control-click) context menu.
+
+- Bitcoin Core 0.20.2rc1 and 0.19.2rc1 are expected to be
+  [available][bitcoincore.org/bin] sometime after the publication of
+  this newsletter.  They contain several bug fixes, such as an
+  improvement described in [Newsletter #110][news110 bcc19620] that will
+  prevent them from redownloading future taproot transactions that they
+  don't understand.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #20564][] makes two changes to the way that Bitcoin Core
+  signals support for `addrv2` messages ([BIP155][]):
+
+  - *Protocol version:* Bitcoin Core will only negotiate support for `addrv2` messages with peers that signal
+    they are using P2P version 70016 or higher. This restriction isn't required by BIP155,
+    but release testing has revealed that some other implementations will disconnect
+    from Bitcoin Core if they receive any unknown message, including
+    `sendaddrv2`. This change may be reverted in future versions of Bitcoin Core,
+    so other implementations are advised to tolerate unknown P2P messages at any time in the connection.
+
+  - *Updated BIP:* The `sendaddrv2` message will now be sent between the `version` and
+    `verack` message, as required by the latest version of BIP155. See [BIPs
+    #1043](#bips-1043) below for more information about that change to the BIP.
+
+    This PR was back-ported to [the latest V0.21 release candidate][Bitcoin
+    Core 0.21.0] in [Bitcoin Core #20612][].
+
+- [Bitcoin Core #19776][] updates the `getpeerinfo` RPC with two new
+  fields. `bip152_hb_to` indicates that we asked the peer to relay new
+  blocks to us by sending a [BIP152][] compact block without waiting to
+  ask if we need that specific block, called High-Bandwidth (HB) mode.
+  `bip152_hb_from` indicates that the peer asked us to be one of their
+  high-bandwidth peers.  By default, each node selects [up to 3][hb
+  peers] high-bandwidth compact block peers.  (Despite the name,
+  high-bandwidth mode doesn't use much bandwidth compared to legacy block
+  relay; its optimizations for fast relay of new blocks just uses more
+  bandwidth than BIP152's low-bandwidth mode.)
+
+- [Bitcoin Core #19858][] adds a new method for finding high-quality peers with
```

Comment:
Note: this is really familiar, because we covered it last week for the PR review club.

Reply:
But I *did* try to improve it :smiley:

Reply:
Well, I looked up whether we had covered the merge last week already, because I felt I had read it before, but realized that it was the PR Review Club summary. Just trying to save others that investigation. :+1: